lab_3/target_score.txt

```diff
@@ -1 +1 @@
-6
\ No newline at end of file
+4
```

Comment:
Do not forget to change it - I see that you have done the n-grams.

Reply:
thanks! i really forgot about it, now everything is changed